### PR TITLE
Style Checking Storage.Blobs

### PR DESCRIPTION
**Pull request by chidozieononiwu**

### CODE REVIEW

**Comment by chidozieononiwu** [comment]
> Can we exclude `/Generated/*RestClient.cs`? Those shouldn't be changed because they'll be blown over on the next regeneration.

I have excluded that.

File: sdk/storage/Azure.Storage.Blobs/src/Sas/BlobAccountSasPermissions.cs [hunk 1]
```diff
@@ -92,20 +105,20 @@ public static BlobAccountSasPermissions Parse(string s)
...
         [EditorBrowsable(EditorBrowsableState.Never)]
         public override int GetHashCode() =>
-            (this.Read   ? 0b000001 : 0) +
-            (this.Add    ? 0b000010 : 0) +
-            (this.Create ? 0b000100 : 0) +
-            (this.Write  ? 0b001000 : 0) +
-            (this.Delete ? 0b010000 : 0) +
-            (this.List   ? 0b100000 : 0);
+            (Read ? 0b000001 : 0) +
+            (Add ? 0b000010 : 0) +
```
Comment (tg-msft) [inline sdk/storage/Azure.Storage.Blobs/src/Sas/BlobAccountSasPermissions.cs]: Does it complain about this whitespace?  Or were we being aggressive?

**Review by tg-msft — APPROVED** [review]
Awesome!  Thank you Dozie!

File: sdk/storage/Azure.Storage.Blobs/src/Sas/BlobAccountSasPermissions.cs (hunk 1 above)
Comment (chidozieononiwu) [reply sdk/storage/Azure.Storage.Blobs/src/Sas/BlobAccountSasPermissions.cs]: Its due to this rule `csharp_space_around_declaration_statements = do_not_ignore`

File: sdk/storage/Azure.Storage.Blobs/samples/Sample02_Auth.cs [hunk 2]
```diff
@@ -169,7 +169,7 @@ public async Task SharedAccessSignatureAuthAsync()
 
             // Try to create a new container (which is beyond our
             // delegated permission)
-            StorageRequestFailedException ex = 
+            StorageRequestFailedException ex =
```
Comment (seanmcc-msft) [inline sdk/storage/Azure.Storage.Blobs/samples/Sample02_Auth.cs]: Would anyone be opposed to adding a check style rule for starting lines with the operator?  I think it's more readable.

```cshare
  StorageRequestFailedException ex
      = Assert.ThrowsAsync<StorageRequestFailedException>(
async () => await service.CreateBlobContainerAsync(Randomize("sample-container")));
```

File: sdk/storage/Azure.Storage.Blobs/src/BlobBaseClient.cs [hunk 3]
```diff
@@ -634,20 +634,20 @@ protected virtual BlobBaseClient WithSnapshotImpl(string snapshot)
...
                 range.Offset + startOffset,
                 range.Count.HasValue ?
                     range.Count.Value - startOffset :
                     (long?)null);
 
-            this.Pipeline.LogTrace($"Download {this.Uri} with range: {pageRange}");
+            Pipeline.LogTrace($"Download {Uri} with range: {pageRange}");
 
-            var response =
+            Response<FlattenedDownloadProperties> response =
```
Comment (seanmcc-msft) [inline sdk/storage/Azure.Storage.Blobs/src/BlobBaseClient.cs]: +1 I like the rule requiring using types instead of var.

**Review by seanmcc-msft — APPROVED** [review]

**Pull request merged**